models/src/transactions.rs

```diff
@@ -84,10 +85,37 @@ impl Transaction {
             .load(conn)
     }
 
+    pub fn get_swap_totals(conn: &PgConnection) -> QueryResult<Vec<SwapTotals>> {
```

Comment:
With this one, user should also get karma when they receive or send payments in fiat to outside wallets. This is because these actions cause a trade on Kollider exchange and hence karma accumulation.